### ISSUE TITLE
Feature Request: Create #kops Slack Channel on kubernetes.slack.com

### ISSUE THREAD

**Issue by osterman** [kubernetes/kops#1401]
## what
* The kubernetes org on slack should have #kops channel

## why
* facilitate community exchange
* most other top level projects have a team (e.g. #helm, #kubeadm, #minikube)

**Comment by chrislovecnm** [comment]
@sarahnovotny / @justinsb / @kris-nova thoughts?

**Comment by justinsb** [comment]
kops is an installation tool, and therefore most discussions seem to overlap with something else.  We should maybe have #kops-dev.  But nobody asks "how do I use kops to do X in kops", they are always asking "how should I do authentication, I am using kops" => sig-auth, or "how should I isolate tenants on my network, I am using kops" => sig-network, or "I'm hitting AWS rate limits, I'm using kops" => sig-aws. 

That's my theory anyway :-)  We don't want sig-auth to not have visibility into how people are struggling to set up auth, because all those problems are hidden (from them) in some slack channel they don't follow.

If we do have kops-users questions, we should have a channel, but then I also don't want to send users on a ping-pong between channels.

When we replace Slack, I think we should try Wave - I think it had a much stronger topic model that was much more real-world :-)

**Comment by chrislovecnm** [comment]
>  I think we should try Wave

You went there ... :man_facepalming: 

> But nobody asks "how do I use kops to do X in kops"

I have definitely had those questions.  But I get you point.  Once we have GCE and baremetal support, I think we will need our own channel.  At this point, I am on the fence.

@osterman can you provide more details?

**Comment by osterman** [comment]
> kops is an installation tool, and therefore most discussions seem to overlap with something else. 
While the root cause of a problem could very likely be somewhere else, I wouldn't expect end users to be as sophisticated. I wouldn't know where to being navigating the channels down to SIG unless I were actively working on the internals. More over, I wouldn't expect users in SIG groups to have any experience with kops or the internals therein.

The fact that #kubeadm, #minikube both have channels I think adds credibility. 

> We don't want sig-auth to not have visibility into how people are struggling to set up auth, because all those problems are hidden (from them) in some slack channel they don't follow.

We don't want kops losing sight of the problems users are having either or how they are trying to use kops ;)

> We should maybe have #kops-dev. 
I guess start with the smallest group that could be useful. Can always break it up in to smaller groups. Consider it a trial. See if it gains traction. If not, can always shutter it. I for one would love to meet others who are using kops. 

> @osterman can you provide more details?

sure, let me give you a case in point.

I'm following the instructions for rotating the ssh pub key. When ever I do the rolling-update (kops 1.4.3), the machines come up fine, but DNS is not updated. If I tail the `dns-controller` I see a bunch of timeouts. If I bring up the cluster from scratch, DNS works. I see there's a slew of open issues related to DNS, so before I open up yet another, thought I would ask to make sure I'm not doing something wrong.

https://github.com/kubernetes/kops/blob/master/docs/security.md#ssh-access
https://github.com/kubernetes/kops/issues?utf8=✓&q=is%3Aissue%20is%3Aopen%20dns

**Comment by osterman** [comment]
(looks like upgrading to 1.4.4 fixed my issue. =) no more dns problems.)

**Comment by kris-nova** [comment]
So here is my $0.02

### It should be simple

One of the big attractions to Kubernetes other than the tool itself is the amazing community. Having a `#kops` or a `#kops-dev` channel could help keep the community connected, especially for newcomers to the project. Over the past few months I have watched kops grow in attention, and it seems like almost every day I see a new user joining `#sig-aws` and then immediately posting a `kops` question.

Having a general `#kops` or `#kops-dev` channel could be a **great** starting place for these new users. Which would allow for introductions, and a safe space for users who don't know (or are intimidated) about the other channels to ask a question. Others could politely answer, and redirect if needed. 

I don't think a `#kops` or `#kops-dev` channel should exist to **replace** channels like `#sig-network` or `#sig-auth` but I do however think the `#kops` channel(s) should exist to introduce people to channels like `#sig-network` or `#sig-auth` when needed. In @osterman's case - the dialog could very well have started in the `#kops` channel, and evolved from there. Crossing into other channels as needed. I see this case more than any other.

Bottom line. The community is driven by feedback, and we should allow users an easy way to provide feedback. 

### It would be great for maintainers, and contributors

I have noticed that we have an ever growing need for communication and organization as maintainers. We already have a few private threads going for kops maintainers, and frankly it would be nice to have a place where we could have a gathering of maintainers and kops enthusiasts. 

Perfect example was the question I posed this morning in private of `bastion.` vs `bastion-`

### Bugs

What about bug reports, we get those **a lot** in other channels. I can almost guarantee if you scroll up in `#sig-aws` at any given time, you will be able to find some sort of dialogue around a bug or a feature and the way it works (or allegedly doesn't). 

When we are tracking down bugs, time can certainly be of the essence and having a **command station** makes sense. 

### User Experience

This is a huge part of kops, we cater towards admins and cluster operators. We need to interact with them, and see how they are using the tool. I wonder how much dialogue we are missing out on just because we don't have a clear space defined for users to discuss UX in kops.

We have had a few of our die-hard contributors open issues around things like having a GUI for kops.. but that was a one sided and potentially slow process to start.

Having a place to have discussions around UX is a great way to turn **good software** into **great software**

```
</endrant>
```

### TLDR;

+1 for a `#kops` channel

**Comment by WillPlatnick** [comment]
+1 for #kops as well

**Comment by chrislovecnm** [comment]
@sarahnovotny any wisdom here?

**Comment by sarahnovotny** [comment]
done 🥇
